docker-compose.yml

```diff
@@ -0,0 +1,146 @@
+  depends_on:
+    - zookeeper
+    - mesos-master
+  links:
+    - zookeeper:zookeeper
+    - mesos-master:master.mesos
+
+services:
+  zookeeper:
+    image: zookeeper:3.4.11
+    hostname: zookeeper
+    ports:
+      - "2181:2181"
+    networks:
+      app_net:
+        ipv4_address: 172.16.190.2
+
+  mesos-master:
+    build:
+      context: ${MESOS_SOURCES}
+      dockerfile: $PWD/Dockerfile.automake
+    image: criteo/mesosbuild
+    command: /usr/bin/mesos-master
+    hostname: localhost
+    ports:
+      - "5050:5050"
+    networks:
+      app_net:
+        ipv4_address: 172.16.190.3
+    links:
+      - zookeeper
+    depends_on:
+      - zookeeper
+    environment:
+      MESOS_ZK: zk://zookeeper:2181/mesos
+      MESOS_CLUSTER: docker-compose
+      MESOS_HOSTNAME: localhost
+      MESOS_WORK_DIR: /var/tmp/mesos
+      MESOS_LOG_DIR: /var/log/mesos
+      LIBPROCESS_IP: 172.16.190.3
+    volumes:
+      - ${MESOS_SOURCES}:/src/mesos/
+
+  marathon-service:
+    image: mesosphere/marathon:v1.10.17
+    ports:
+      - "8080:8080"
+    hostname: localhost
+    networks:
+      app_net:
+        ipv4_address: 172.16.190.5
+    links:
+      - zookeeper
+      - mesos-master
+    depends_on:
+      - zookeeper
+      - mesos-master
+    environment:
+      - MARATHON_ZK=zk://zookeeper:2181/marathon
+      - MARATHON_MASTER=zk://zookeeper:2181/mesos
+      - LIBPROCESS_IP=172.16.190.5
+
+  mesos-slave-1:
+    << : *mesos-slave
+    hostname: mesos-slave-1
+    ports:
+      - "5051:5051"
+    environment:
+      MESOS_PORT: 5051
+      MESOS_HOSTNAME: mesos-slave-1
+      LIBPROCESS_IP: 172.16.190.10
+    networks:
+      app_net:
+        ipv4_address: 172.16.190.10
+
+  mesos-slave-2:
+    << : *mesos-slave
+    hostname: mesos-slave-2
+    ports:
+      - "5052:5052"
+    environment:
+      MESOS_PORT: 5052
+      MESOS_HOSTNAME: mesos-slave-2
+      LIBPROCESS_IP: 172.16.190.11
+    networks:
+      app_net:
+        ipv4_address: 172.16.190.11
+
+  mesos-slave-3:
+    << : *mesos-slave
+    hostname: mesos-slave-3
+    ports:
+      - "5053:5053"
+    environment:
+      MESOS_PORT: 5053
+      MESOS_HOSTNAME: mesos-slave-3
+      LIBPROCESS_IP: 172.16.190.12
+    networks:
+      app_net:
+        ipv4_address: 172.16.190.12
+
+  mesos-slave-4:
+    << : *mesos-slave
+    hostname: mesos-slave-4
+    ports:
+      - "5054:5054"
+    environment:
+      MESOS_PORT: 5054
+      MESOS_HOSTNAME: mesos-slave-4
+      LIBPROCESS_IP: 172.16.190.13
+    networks:
+      app_net:
+        ipv4_address: 172.16.190.13
+
+  mesos-slave-5:
+    << : *mesos-slave
+    hostname: mesos-slave-5
+    ports:
+      - "5055:5055"
+    environment:
+      MESOS_PORT: 5055
+      MESOS_HOSTNAME: mesos-slave-5
+      LIBPROCESS_IP: 172.16.190.14
+    networks:
+      app_net:
+        ipv4_address: 172.16.190.14
+
+volumes:
+  dist: {}
+
+networks:
+  app_net:
+    driver: bridge
+    ipam:
+      driver: default
+      config:
+      -
```

Comment:
typo?

Reply:
Shamelessly took that from marathon-ui's docker-compose; I think it defines an unamed network entry with just subnet info.